google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerExceptionFactoryTest.java

```diff
@@ -250,4 +250,17 @@ private Metadata createResourceTypeMetadata(String resourceType, String resource
 
     return trailers;
   }
+
+  @Test
+  public void withRequestId() {
+    XGoogSpannerRequestId reqIdIn = XGoogSpannerRequestId.of(1, 2, 3, 4);
+    Status status = Status.fromCodeValue(Status.Code.ABORTED.value());
+    Exception exc = new StatusRuntimeException(status);
+    SpannerException spannerExceptionWithReqId =
+        SpannerExceptionFactory.newSpannerException(exc, reqIdIn);
+    assertThat(spannerExceptionWithReqId.getRequestId()).isEqualTo(reqIdIn.toString());
```

Comment:
nit, for following pull requests: Please prefer the use of standard JUnit `assertEquals(expected, actual)` over `assertThat`, even when the test class in question uses `assertThat`.